## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 452881`
Commit: [`b061d5ca060b215358963c4490f6c53ca5bf2f4e`](https://github.com/NixOS/nixpkgs/commit/b061d5ca060b215358963c4490f6c53ca5bf2f4e) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/b061d5ca060b215358963c4490f6c53ca5bf2f4e..pull/452881/head))
Merge: [`7baba6e05b278f67470e75e367546ea1d87872fb`](https://github.com/NixOS/nixpkgs/commit/7baba6e05b278f67470e75e367546ea1d87872fb)

Logs: https://github.com/drawbu/nixpkgs-review-gha/actions/runs/18621807128


---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>mopidy-tidal</li>
    <li>mopidy-tidal.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>high-tide</li>
    <li>python312Packages.tidalapi</li>
    <li>python312Packages.tidalapi.dist</li>
    <li>python313Packages.tidalapi</li>
    <li>python313Packages.tidalapi.dist</li>
    <li>tauon</li>
    <li>tauon.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>mopidy-tidal</summary>
<pre>&gt;       assert tpp.as_list() == [
            Ref(name=&quot;Playlist-101&quot;, type=&quot;playlist&quot;, uri=&quot;tidal:playlist:101&quot;),
            Ref(name=&quot;Playlist-2&quot;, type=&quot;playlist&quot;, uri=&quot;tidal:playlist:2&quot;),
            Ref(name=&quot;Playlist-222&quot;, type=&quot;playlist&quot;, uri=&quot;tidal:playlist:222&quot;),
        ]
E       AssertionError: assert [Ref(name=&#x27;Pl...laylist:101&#x27;)] == [Ref(name=&#x27;Pl...laylist:222&#x27;)]
E         
E         Right contains 2 more items, first extra item: Ref(name=&#x27;Playlist-2&#x27;, type=&#x27;playlist&#x27;, uri=&#x27;tidal:playlist:2&#x27;)
E         Use -v to get more diff

tests/test_playlist.py:345: AssertionError
=============================== warnings summary ===============================
../../nix/store/rpdajwc9vgjpwjx42zcpisrpih81kdsf-mopidy-3.4.2/lib/python3.13/site-packages/mopidy/__init__.py:5
  /nix/store/rpdajwc9vgjpwjx42zcpisrpih81kdsf-mopidy-3.4.2/lib/python3.13/site-packages/mopidy/__init__.py:5: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools&lt;81.
    import pkg_resources

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_playlist.py::test_playlist_sync_downtime - AssertionError: assert [Ref(name=&#x27;Pl...laylist:101&#x27;)] == [Ref(name=&#x27;Pl...la...
======= 1 failed, 177 passed, 10 xfailed, 8 xpassed, 1 warning in 2.17s ========</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>high-tide</li>
    <li>mopidy-tidal</li>
    <li>mopidy-tidal.dist</li>
    <li>python312Packages.tidalapi</li>
    <li>python312Packages.tidalapi.dist</li>
    <li>python313Packages.tidalapi</li>
    <li>python313Packages.tidalapi.dist</li>
    <li>tauon</li>
    <li>tauon.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>mopidy-tidal</li>
    <li>mopidy-tidal.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>python312Packages.tidalapi</li>
    <li>python312Packages.tidalapi.dist</li>
    <li>python313Packages.tidalapi</li>
    <li>python313Packages.tidalapi.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 7 packages built:</summary>
  <ul>
    <li>mopidy-tidal</li>
    <li>mopidy-tidal.dist</li>
    <li>python312Packages.tidalapi</li>
    <li>python312Packages.tidalapi.dist</li>
    <li>python313Packages.tidalapi</li>
    <li>python313Packages.tidalapi.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 452881`
Commit: [`b061d5ca060b215358963c4490f6c53ca5bf2f4e`](https://github.com/NixOS/nixpkgs/commit/b061d5ca060b215358963c4490f6c53ca5bf2f4e) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/b061d5ca060b215358963c4490f6c53ca5bf2f4e..pull/452881/head))
Merge: [`7baba6e05b278f67470e75e367546ea1d87872fb`](https://github.com/NixOS/nixpkgs/commit/7baba6e05b278f67470e75e367546ea1d87872fb)

Logs: https://github.com/drawbu/nixpkgs-review-gha/actions/runs/18621807128


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>high-tide</li>
    <li>mopidy-tidal</li>
    <li>mopidy-tidal.dist</li>
    <li>python312Packages.tidalapi</li>
    <li>python312Packages.tidalapi.dist</li>
    <li>python313Packages.tidalapi</li>
    <li>python313Packages.tidalapi.dist</li>
    <li>tauon</li>
    <li>tauon.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>high-tide</li>
    <li>mopidy-tidal</li>
    <li>mopidy-tidal.dist</li>
    <li>python312Packages.tidalapi</li>
    <li>python312Packages.tidalapi.dist</li>
    <li>python313Packages.tidalapi</li>
    <li>python313Packages.tidalapi.dist</li>
    <li>tauon</li>
    <li>tauon.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>mopidy-tidal</li>
    <li>mopidy-tidal.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>python312Packages.tidalapi</li>
    <li>python312Packages.tidalapi.dist</li>
    <li>python313Packages.tidalapi</li>
    <li>python313Packages.tidalapi.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 7 packages built:</summary>
  <ul>
    <li>mopidy-tidal</li>
    <li>mopidy-tidal.dist</li>
    <li>python312Packages.tidalapi</li>
    <li>python312Packages.tidalapi.dist</li>
    <li>python313Packages.tidalapi</li>
    <li>python313Packages.tidalapi.dist</li>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

--- a/pkgs/development/python-modules/tidalapi/default.nix
+++ b/pkgs/development/python-modules/tidalapi/default.nix
@@ -49,6 +49,7 @@ buildPythonPackage rec {
     homepage = "https://github.com/tamland/python-tidal";
     license = lib.licenses.gpl3;
     maintainers = with lib.maintainers; [
+      drafolin
       drawbu
       ryand56
     ];

--- a/pkgs/development/python-modules/tidalapi/default.nix
+++ b/pkgs/development/python-modules/tidalapi/default.nix
@@ -9,17 +9,18 @@
   ratelimit,
   typing-extensions,
   mpegdash,
+  pyaes,
 }:
 buildPythonPackage rec {
   pname = "tidalapi";
-  version = "0.8.6";
+  version = "0.8.8";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "EbbLabs";
     repo = "python-tidal";
     tag = "v${version}";
-    hash = "sha256-SsyO0bh2ayHfGzINBW1BTTPS/ICvIymIhQ1HUPRFOwU=";
+    hash = "sha256-+O+U8QZhaOdUgPONv1tb5ctiK8NmD2NJK0hokmNtUZM=";
   };
 
   build-system = [
@@ -33,6 +34,7 @@ buildPythonPackage rec {
     isodate
     ratelimit
     typing-extensions
+    pyaes
   ];
 
   doCheck = false; # tests require internet access

